src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -226,10 +226,10 @@ protected function args_to_context( array $arguments, Context $context ) {
 	 *
 	 * @return array<string,mixed> The modified template variables.
 	 */
-	public function disable_json_data( $template_vars ) {
+	public function disable_json_data( $template_vars, $view ) {
```

Comment:
you're not using `$view` in this function, is there a reason why you're adding it?

Reply:
I believe the intent was to allow testing it, but we're not so it can be removed.

Reply:
Removed.